spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -0,0 +1,40 @@
+= Spring Cloud GCP Logging Starter Example
+
+This sample application demonstrates using the
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging[Spring Cloud GCP Logging Starter] in your code.
+
+== Setup & Configuration
+1. Create a Google Cloud Platform Project
+1. Have the https://cloud.google.com/sdk/[Google Cloud SDK] installed and
+initialized and logged in with
+https://developers.google.com/identity/protocols/application-default-credentials[application
+default credentials].
+
+1. Check that the https://cloud.google.com/logging/docs/access-control[Stackdriver Logging API is enabled].
+
+== Run the Example
+Deploy the sample application from Maven to App Engine Flexible Environment:
```

Comment:
Instead of using App Engine to test, can we use curl locally and adding the header?
Would prefer to avoid any connections to App Engine unless they're strictly required.

Reply:
This is a sample though, not a test. It's nice that we can show how this would work with a GAE deployment since we're doing this for GCP.

Reply:
I'm sorry, but I can't approve this PR if it contains this GAE bit.
Can talk more offline if you want.

Reply:
Actually, using Spring Cloud GCP logging with GAE Flex (or Standard) sounds great for a codelab at g.co/codelabs/spring.

Reply:
Could you explain why you feel strongly against having it in?